<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dostepne_Produkty</class>
 <widget class="QDialog" name="Dostepne_Produkty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ZmienicIloscButton">
      <property name="text">
       <string>Zmienić ilość</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EdytowacButton">
      <property name="text">
       <string>Edytować Produkt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="InfoButton">
      <property name="text">
       <string>Produkt Info</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>81</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
